lit_tests/kernel/wave/codegen.py

```diff
@@ -498,7 +500,7 @@ def test(
         )
         tkw.write(res, b, elements_per_thread=16)
 
-    with codegen_test_context(canonicalize=True):
+    with codegen_test_context(canonicalize=True, additional_symbols={ONE: 1}):
```

Comment:
why add ONE as an additional symbol? Is it to avoid adding it to the hyperparams every time?